build_lxc_static_resources_image.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+source ./_common.sh
+
+function build_docker_image {
+	local base_image_version=$(./release_notes.sh get-version)
+	local lxc_static_resources_image_version=0.1.0
+	local liferay_docker_org=${LIFERAY_DOCKER_ORG:-liferay}
```

Comment:
Where does $LIFERAY_DOCKER_ORG gets its value? If its provided by the user, please add the variable to the **build_bundle_image.sh -> check_usage** function.

Reply:
I removed that part instead and went with hard coded `liferay` org like every other use case.